Main.sublime-menu

```diff
@@ -0,0 +1,49 @@
+[
+	{
+		"caption": "Tools",
+		"id": "tools",
+		"children": [
+			{
+				"id": "SublimeLinter-contrib-xo",
+				"caption": "SublimeLinter XO",
+				"children": [
+					{
+						"caption": "Fix current file",
+						"command": "xo_fix"
+					}
+				]
+			}
+		]
+	},
+	{
+		"caption": "Preferences",
+		"id": "preferences",
+		"children": [
+			{
+				"caption": "Package Settings",
+				"id": "package-settings",
+				"children": [
+					{
+						"caption": "SublimeLinter XO",
+						"children": [
+							{
+								"caption": "Settings - Default",
+								"command": "open_file",
+								"args": {
+									"file": "${packages}/SublimeLinter-contrib-xo/SublimeLinterContribXO.sublime-settings"
```

Comment:
Is there really no variable for the full path to the plugin or the sublime-settings file?

Reply:
It seems not.
https://www.sublimetext.com/docs/3/build_systems.html#variables
http://gohom.win/ManualHom/IDE/Sublime/sublime-text-unofficial-documentation-3.0/#build-system-variables